src/zkevm_specs/evm/execution/extcodecopy.py

```diff
@@ -0,0 +1,60 @@
+from ..instruction import Instruction, Transition
+from ...util.param import EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS
+from ...util import N_BYTES_MEMORY_ADDRESS, FQ
+from ..table import AccountFieldTag, CallContextFieldTag, CopyDataTypeTag
+
+
+def extcodecopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    address_word, memory_offset_word, code_offset_word, size_word = (
+        instruction.stack_pop(),
+        instruction.stack_pop(),
+        instruction.stack_pop(),
+        instruction.stack_pop(),
+    )
+
+    address = instruction.rlc_to_fq(address_word, 20)
```

Comment:
```suggestion
```

Reply:
Fixed in 788e85ba9f13075420111460fefb9e6b9e5e5bfe :)